packages/evm-adapter/src/constants.ts

```diff
@@ -1,4 +1,12 @@
 import { BigNumber, utils } from 'ethers';
 
-export const contractAddress = '0x707Cad6EDB79eE9F64818B09F14A7Aa7f4910e93';
+// For compability with BE and FE, remove in future
+export const contractAddress = '0xa35e123aA4AFeD2E4957897232F2ABf73f34cfB0';
```

Comment:
don't you want to use this variable inside ContractAddresses ?
```TS
export enum ContractAddresses {
  POLYGON_MUMBAI = contractAddress,
  POLYGON_MAINNET = contractAddress,
}
```

Reply:
Removed